koku/api/report/test/ocp_aws_static_data.yml

```diff
@@ -4,135 +4,121 @@ generators:
               volumes:
-                - volume:
-                  volume_name: pvc-volume_1
+                - volume_name: pvc-volume_1
                   storage_class: gp2
                   volume_request_gig: 20
-                  labels: label_environment:dev|label_app:catalog|label_version:prod|label_storageclass:bravo
+                  labels: environment:dev|app:catalog|version:prod|storageclass:bravo
                   volume_claims:
-                  - volume_claim:
-                    volume_claim_name: pod_name1_data
+                  - volume_claim_name: pod_name1_data
                     pod_name: pod_name1
-                    labels: label_environment:dev|label_app:catalog|label_version:prod|label_storageclass:bravo
+                    labels: environment:dev|app:catalog|version:prod|storageclass:bravo
                     capacity_gig: 20
-            catalog:
+            - namespace_name: catalog
               pods:
-                - pod:
-                  pod_name: pod_name2
+                - pod_name: pod_name2
                   cpu_request: 1
                   mem_request_gig: 2
                   cpu_limit: 1
                   mem_limit_gig: 4
                   pod_seconds: 3600
-                  labels: label_environment:dev|label_app:catalog|label_version:prod
+                  labels: environment:dev|app:catalog|version:prod
               volumes:
-                - volume:
-                  volume_name: pvc-volume_2
+                - volume_name: pvc-volume_2
                   storage_class: gp2
                   volume_request_gig: 20
-                  labels: label_environment:dev|label_app:catalog|label_version:prod|label_storageclass:delta
+                  labels: environment:dev|app:catalog|version:prod|storageclass:delta
                   volume_claims:
-                  - volume_claim:
-                    volume_claim_name: pod_name2_data
+                  - volume_claim_name: pod_name2_data
                     pod_name: pod_name2
-                    labels: label_environment:dev|label_app:catalog|label_version:prod|label_storageclass:delta
+                    labels: environment:dev|app:catalog|version:prod|storageclass:delta
                     capacity_gig: 20
-        - node:
-          node_name: aws_compute_2
+        - node_name: aws_compute_2
           cpu_cores: 2
           memory_gig: 8
-          resource_id: 55555556
+          resource_id: i-55555556
+          labels: environment:qe
           namespaces:
-            cost-management:
+            - namespace_name: cost-management
               pods:
-                - pod:
-                  pod_name: pod_name3
+                - pod_name: pod_name3
                   cpu_request: 1
                   mem_request_gig: 2
                   cpu_limit: 1
                   mem_limit_gig: 4
                   pod_seconds: 3600
-                  labels: label_environment:qe|label_app:cost|label_version:beta
```

Comment:
These had `label_` on them because that's how our real data looks like coming from OpenShift. I think in dropping this we drift farther from a real report.

Reply:
Ok. I can add it back, but it needs to match the labels in the AWS tag data. So, the AWS tags also need to include `label_`, which they didn't.

As far as I can tell, Masu wasn't stripping the `label_` bit off and the data-as-is wasn't lining up for the ocp-on-<cloud> stuff to work.

So, there's some behavior here that probably needs documenting, if you are able to talk through what it's supposed to be.

Reply:
done!